examples/ecto_job_priority_demo/lib/ecto_job_priority_demo/job_monitor.ex

```diff
@@ -0,0 +1,59 @@
+defmodule EctoJobPriorityDemo.JobMonitor do
+  @moduledoc false
+  use GenServer
+
+  alias EctoJobPriorityDemo.JobQueue
+  alias EctoJobPriorityDemo.Repo
+
+  def start_link(jobs \\ %{count: 1, priority: 0, period: 1000}, server) do
+    GenServer.start_link(__MODULE__, jobs, name: server)
+  end
+
+  def init(%{count: count, priority: priority, period: period}) do
+    send(self(), {:produce_jobs, count, priority, period})
+    {:ok, 0}
+  end
+
+  def update(server, value) do
+    GenServer.cast(server, {:update, value})
+  end
+
+  def count(server) do
+    GenServer.call(server, :count)
+  end
+
+  # Server
+
+  def handle_cast({:update, value}, state) do
+    {:noreply, state + value}
+  end
+
+  def handle_call(:count, _from, state) do
+    {:reply, state, state}
+  end
+
+  def handle_info({:produce_jobs, count, priority, period}, state) do
+    jobs =
+      Enum.map(1..count, fn _ ->
```

Comment:
Perhaps it would be more idiomatic to use an `Ecto.Multi` to insert a batch of jobs?

Reply:
@mbuhot Done!